### ISSUE TITLE
Error with CurrencyExchange deserialization

### ISSUE THREAD

**Issue by AdamTovatt** [RobinTTY/NordigenApiClient#6]
I'm using the AccountsEndpoint.GetTransactions(string id) function but it throws an exception:

```
System.Runtime.Serialization.SerializationException: Deserialization failed, please report this issue to the library author: https://github.com/RobinTTY/NordigenApiClient/issues
The following JSON content caused the problem:  [json]
```
```
System.Text.Json.JsonException: The JSON value could not be converted to RobinTTY.NordigenApiClient.Models.Responses.CurrencyExchange. Path: $.transactions.booked[2].currencyExchange | LineNumber: 0 | BytePositionInLine: 1084.
```

I don't want to post the whole json here since it contains all my bank transactions but I think this might be the part of the json that causes it:
```
"currencyExchange":[{"sourceCurrency":"USD","exchangeRate":"10.3373000","unitCurrency":"USD","targetCurrency":"SEK"}]
```

**Comment by RobinTTY** [comment]
Hm interesting, in the past the currency exchange was not an array...
I'll fix this later today, thanks for reporting!

**Comment by RobinTTY** [comment]
Should be fixed with v6.1.2. Would be great if you could confirm once you get to it :)

**Comment by AdamTovatt** [comment]
Yes, it works with the new version (v6.1.2) of the package! Thank you for the fix and thank you for the package in general! 🥇 👍
